cmd/lock-rpc-server.go

```diff
@@ -20,7 +20,7 @@ import (
 	"fmt"
 	"math/rand"
 	"net/rpc"
-	"path"
+	pathutil "path"
```

Comment:
It happens in posix.go but it is also naturally possible to use a local variable by someone . which in turn generates this problem. I can keep it `path` for now.